spyder/plugins/editor/widgets/tests/test_comments.py

```diff
@@ -62,30 +62,30 @@ def test_single_line_comment(codeeditor):
     # Toggle comment with space at the right of prefix but manually inserted
     text = toggle_comment(editor, start_line=2)
     assert text == ("class a():\n"
-                    "    self.b = 1\n"
```

Comment:
Why is this space not present now? I think your changes shouldn't alter this, so I don't understand this change.

The same goes for all modifications you did for all other tests here.

Reply:
The text line 35 is defined with Spyder 3 style comments (leading `#` with no space) but the tests are expecting Spyder 4/5/6 style comments (`# ` with a space).

This PR breaks compatibility with Spyder 3 style comments which I have argued elsewhere in the comments is prefered (in my opinion) to breaking the formatting after commenting/uncommenting. In this case we could rewrite the tests to use Spyder 4/5/6 style comments.